README.md

```diff
@@ -12,78 +16,94 @@ Some paragraphs are considered "internal", and only meant to be used inside othe
+## Usage as a dependency in another project
 
-## Installation
-
-The recommended way of installing the OpenEuropa Whitelabel Theme is via [Composer](https://www.drupal.org/docs/develop/using-composer/using-composer-to-manage-drupal-site-dependencies#managing-contributed).
-
-```bash
-composer require openeuropa/oe_whitelabel
-```
+Website projects can use `oe_whitelabel` either as an active theme, or they can create a custom theme using `oe_whitelabel` as a base theme.
 
-### Enable the theme
+### Requirements
 
-In order to enable the theme in your project perform the following steps:
+The package is meant for Drupal projects that manage their dependencies with [Composer](https://www.drupal.org/docs/develop/using-composer/using-composer-to-manage-drupal-site-dependencies#managing-contributed).
 
-- Enable the OpenEuropa Whitelabel Theme and set it as default ```./vendor/bin/drush config-set system.theme default oe_whitelabel_theme```
+Ideally this project should be managed with [Docker](https://www.docker.com/get-docker) and [Docker Compose](https://docs.docker.com/compose/), but this is not a hard requirement.
 
-### Integration with oe_paragraphs
+Check the [composer.json](composer.json) for required PHP version and other dependencies.
 
-In order to have full working integration with paragraphs in your project, you must enable oe_whitelabel_paragraphs module:
+### Add the composer package
 
-```./vendor/bin/drush en oe_whitelabel_paragraphs```
+Add this manually in composer.json, or combine with existing entries:
 
-## Development setup
+```
+    "extra": {
+        "artifacts": {
+            "openeuropa/oe_bootstrap_theme": {
+                "dist": {
+                    "url": "https://github.com/{name}/releases/download/{pretty-version}/{project-name}-{pretty-version}.zip",
+                    "type": "zip"
+                }
+            },
+            "openeuropa/oe_whitelabel": {
+                "dist": {
+                    "url": "https://github.com/{name}/releases/download/{pretty-version}/{project-name}-{pretty-version}.zip",
+                    "type": "zip"
+                }
+            }
+        }
+    }
+```
 
-OpenEuropa Whitelabel theme uses [Webpack](https://webpack.js.org) to compile and bundle SASS and JS.
+If your project is still in development, you may modify composer.json to allow alpha versions, while this package and some of its dependencies are still in their `alpha` phase.
 
-Make sure you have Node and npm installed.
+Review this again when the site goes into production.
 
-You can read a guide on how to install node here: https://docs.npmjs.com/getting-started/installing-node
+```
+    "minimum-stability": "alpha",
+```
 
-If you prefer to use [Yarn](https://yarnpkg.com) instead of npm, install Yarn by following the guide [here](https://yarnpkg.com/docs/install).
+Require with composer:
 
-To install required Node.js dependencies run:
+```bash
+composer require openeuropa/oe_whitelabel
```

Comment:
I just tried this again with an empty project where I only require `openeuropa/oe_whitelabel`.
The goal is a successful `require` command that gets something from `1.0*alpha` and not anything from `0.x`.

- If you have `"prefer-stable": false` and `"minimum-stability": "alpha"`, then you don't even need to specify a version string at all. I think people should avoid doing this.
- If you have `"prefer-stable": true` and `"minimum-stability": "alpha"`, then you need to specify a version string, but it is enough to require `^1.0` to avoid getting `0.x`. The cool thing is this can stay there even once 1.0 beta or stable is released.
- If you have `"prefer-stable": true` and you don't set `"minimum-stability"`, then you need to specify a version string with alpha for whitelabel _and_ oe_bootstrap_theme.

My concern was that some people/projects don't want to set `minimum-stability` because it can impact all packages.

I am trying something, let me know what you think.